.github/workflows/pytest.yml

```diff
@@ -26,9 +26,15 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python_version }}
-      - name: Install package
+      - name: Install uv
         run: |
           curl -LsSf https://astral.sh/uv/install.sh | sh
+      - name: Install nomad
+        if: "${{ matrix.python_version != '3.8' && matrix.python_version != '3.12'}}"
+        run: |
+          uv pip install nomad-lab@git+https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-FAIR.git
+      - name: Install package
+        run: |
           uv pip install ".[dev]"
       - name: Test with pytest
```

Comment:
```suggestion
      - name: Test pynxtools data conversion
        run: |
          pytest tests/test_reader.py
      - name: Test Nomad Examples
        if: "${{ matrix.python_version != '3.8' && matrix.python_version != '3.12'}}"
        run: |
          pytest tests/test_nomad_examples.py
```

Reply:
Otherwise we will get an error as nomad is not available for all the python versions.

Reply:
Availability of the `nomad` is also part of tests, right? For some reasons, if nomad is not available and we can not see inconsistency in our examples.

Reply:
It does not matter where you skip the tests. For all the other plugins, I implemented the skipping in the `test_nomad_examples` because then you only need to check for the python version once in the workflow file. Since the allowed versions may change, this seemed like the cleaner solution to me.